plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/base/BaseFBImplTemplate.xtend

```diff
@@ -59,22 +61,19 @@ abstract class BaseFBImplTemplate<T extends BaseFBType> extends ForteFBTemplate<
 			«type.internalConstVars.generateVariableDefinitions(true)»
 			
 		«ENDIF»
-		«IF !type.internalFbs.isEmpty»
-			«type.generateInternalFbDeclarations»
-			
-		«ENDIF»
+		
 		«FBClassName»::«FBClassName»(const CStringDictionary::TStringId paInstanceNameId, forte::core::CFBContainer &paContainer) :
 		    «baseClass»(paContainer, &scmFBInterfaceSpec, paInstanceNameId, «IF !type.internalVars.empty»&scmInternalVars«ELSE»nullptr«ENDIF»)«// no newline
+		    		    »«generateInternalFBInitializer»«// no newline
```

Comment:
Please fix the indendation to use a tab instead of 4 spaces right before the `»`.

Reply:
Done